packages/core/package.json

```diff
@@ -81,6 +91,7 @@
   },
   "dependencies": {
     "@remote-dom/polyfill": "workspace:^1.4.2",
+    "@remote-ui/core": "^2.0.0",
```

Comment:
What do you think about an optional peer dependency for this instead? You'd need to also add it as a dev dependency (so types can be checked), but I think it might send the wrong message to have the new library explicitly depend on the old one like this, even if it’s for a good reason.

Reply:
After a round of discussion we decided to extract it into a separate package. I went with `@remote-dom/compat` name.